frontend/util/Bar.js

```diff
@@ -0,0 +1,99 @@
+import React, { useEffect, useRef, useState } from 'react';
+import * as d3 from 'd3';
+
+const colors = d3.scaleOrdinal(d3.schemeCategory10);
+const format = d3.format('.2f');
+
+const XAxis = ({ top, bottom, left, right, height, scale }) => {
+  const axis = useRef(null);
+
+  useEffect(() => {
+    d3.select(axis.current).call(d3.axisBottom(scale));
+  });
+
+  return <g className="axis x" ref={axis} transform={`translate(${left}, ${height - bottom})`} />;
+};
+
+const YAxis = ({ top, bottom, left, right, scale }) => {
+  const axis = useRef(null);
+
+  useEffect(() => {
+    d3.select(axis.current).call(d3.axisLeft(scale));
+  });
+
+  return <g className="axis y" ref={axis} transform={`translate(${left}, ${top})`} />;
+};
+
+const Rect = ({ data, x, y, height, top, bottom }) => {
+  return (
+    <g transform={`translate(${x(data.date)}, ${y(data.value)})`}>
+      <rect
+        width={x.bandwidth()}
+        height={height - bottom - top - y(data.value)}
+        fill={colors(data.index)}
+      />
+      <text
+        transform={`translate(${x.bandwidth() / 2}, ${-2})`}
+        textAnchor="middle"
+        alignmentBaseline="baseline"
+        fill="grey"
+        fontSize="10"
+      >
+        {format(data.value)}
+      </text>
+    </g>
+  );
+};
+
+const Bar = props => {
```

Comment:
Why not `export default Bar ...`?

Reply:
Bar.js removed , using line chart for showing the historic change of camera load